src/lib/Behat/BusinessContext/UserNotificationContext.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\BusinessContext;
+
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\ElementFactory;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\UpperMenu;
+use Behat\Gherkin\Node\TableNode;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\UserNotificationPopup;
+use PHPUnit\Framework\Assert;
+
+class UserNotificationContext extends BusinessContext
+{
+    /**
+     * @Given there is an unread notification for current user
+     */
+    public function thereIsNotificationForCurrentUser()
+    {
+        $upperMenu = ElementFactory::createElement($this->utilityContext, UpperMenu::ELEMENT_NAME);
+        Assert::assertGreaterThan(0, $upperMenu->getNotificationsCount());
+    }
+
+    /**
+     * @Given I go to item with details:
```

Comment:
I don't like this step - there's nothing here that would say that it'll be done through notification popup. It'll be confusing in `feature` files.